### ISSUE TITLE
20A DC motor control

### ISSUE THREAD

**Issue by rusefillc** [rusefi/hellen125honda-issues#24]
Tommy says "fancy aftermarket wastegates need 20A H-bridges"

@blundar says "20A? That you can actually buy? Nope. Predriver and discrete mosfets"

**Comment by rusefillc** [comment]
@mck1117 @dron0gus FYI

**Comment by mck1117** [comment]
> @blundar says "20A? That you can actually buy? Nope. Predriver and discrete mosfets"

yeah, there do exist chips that otherwise behave like a tle9201 (or whatever), but use 4x external mosfets
